oauthenticator/oauth2.py

```diff
@@ -663,6 +681,19 @@ def _allowed_scopes_validation(self, proposal):
         """,
     )
 
+    pkce = Bool(
+        os.environ.get("OAUTH2_PKCE", "False").lower() in {"true", "1"},
```

Comment:
I don't think we need this environment variable, it's present in some of the other config options for backwards compatibility.

Reply:
Removed.